common/transport/http/test/_rest_api_client_test.js

```diff
@@ -113,6 +113,23 @@ describe('RestApiClient', function() {
       client.executeApiCall('GET', '/fake/path', null, null, testCallback);
     });
 
+    /* Tests_SRS_NODE_IOTHUB_REST_API_CLIENT_18_002: [ If an `x509` cert was passed into the constructor via the `config` object, `executeApiCall` shall use it to establish the TLS connection. ] */
+    it('uses the x509 cert if passed', function(testCallback) {
+      var fakeHttpHelper = {
+        buildRequest: function(method, path, headers, host, x509, requestCallback) {
```

Comment:
>function [](start = 22, length = 8)

super minor nitpick: i've read in style guides that for readability anonymous functions should have a space between 'function' and the opening '('. we've been inconsistent so far (and it's not a big issue) but I'd like us to slowly improve consistency by using 'function ()' with a space between 'function' and '()'